src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/config.jelly

```diff
@@ -32,6 +32,9 @@ THE SOFTWARE.
         <f:entry field="id" title="${%Custom ID}">
             <f:textbox/>
         </f:entry>
+        <f:entry field="cancel" title="${%Cancel Button Caption}">
+            <f:textbox/>
```

Comment:
No `default`, yet `getCancel` is setting a default. I think this means that snippet generator will suggest a `cancel` parameter even if you did not touch this option, which is undesirable: parameters proposed for the Groovy form of the step should include solely those fields which you actually edited from the default. [Docs](https://www.jenkins.io/doc/developer/plugin-development/pipeline-integration/#handling-default-values)

Reply:
That doesn't seem to be the case. I have deployed my custom HPI to our jenkins instance, and the behaviour is as following:
default when selecting the "input" sample step and clicking "Generate pipeline script"
![image](https://github.com/user-attachments/assets/f193387e-ceb2-47b2-a94f-d7fb7d9b0e2d)
after expanding "advanced" and clicking "generate pipeline script"
![image](https://github.com/user-attachments/assets/15b7aa7f-08b9-4934-addd-8a747f09acfd)
Adding a custom "cancel" caption
![image](https://github.com/user-attachments/assets/a2e4cb31-15e0-45fc-9182-55a472889c73)
Adding a custom "OK" caption
![image](https://github.com/user-attachments/assets/e1f9fdef-dbec-4ef7-8a21-853757fcf73e)
Adding both
![image](https://github.com/user-attachments/assets/45c0ab82-5be1-4476-89aa-e2bbb53546f4)